src/base/bittorrent/session.cpp

```diff
@@ -4719,16 +4719,16 @@ void Session::handleSessionStatsAlert(const lt::session_stats_alert *p)
 
     m_status.hasIncomingConnections = static_cast<bool>(stats[m_metricIndices.net.hasIncomingConnections]);
 
-    const auto ipOverheadDownload = stats[m_metricIndices.net.recvIPOverheadBytes];
-    const auto ipOverheadUpload = stats[m_metricIndices.net.sentIPOverheadBytes];
-    const auto totalDownload = stats[m_metricIndices.net.recvBytes] + ipOverheadDownload;
-    const auto totalUpload = stats[m_metricIndices.net.sentBytes] + ipOverheadUpload;
-    const auto totalPayloadDownload = stats[m_metricIndices.net.recvPayloadBytes];
-    const auto totalPayloadUpload = stats[m_metricIndices.net.sentPayloadBytes];
-    const auto trackerDownload = stats[m_metricIndices.net.recvTrackerBytes];
-    const auto trackerUpload = stats[m_metricIndices.net.sentTrackerBytes];
-    const auto dhtDownload = stats[m_metricIndices.dht.dhtBytesIn];
-    const auto dhtUpload = stats[m_metricIndices.dht.dhtBytesOut];
+    const long ipOverheadDownload = stats[m_metricIndices.net.recvIPOverheadBytes];
+    const long ipOverheadUpload = stats[m_metricIndices.net.sentIPOverheadBytes];
```

Comment:
Maybe these should stay 'auto'? Because `stats` type is determined by libt.

Reply:
As discussed above, every one of these will go back to `auto`, like originally proposed.

Reply:
Sorry i missed it.